tools/fileinfo/features/pe-loader-corruptions/test.py

```diff
@@ -462,3 +462,16 @@ def test_corrupted_pe(self):
         self.assertEqual(self.fileinfo.output["loaderError"]["code"], 41)
         self.assertEqual(self.fileinfo.output["loaderError"]["code_text"], 'LDR_ERROR_RSRC_OVER_END_OF_IMAGE')
         self.assertEqual(self.fileinfo.output["loaderError"]["loadable_anyway"], 'true')
+
```

Comment:
Could you please rewrite the commit message so it does not contain a link to an internal bug tracker?
```
git checkout LZ_RETDEC_7
git commit --amend
# Rewrite the commit message, save and exit.
git push --force
```